### PR TITLE
data: wyntonfranklin

### PR DESCRIPTION
**Pull request by wyntonfranklin**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3989"></a>

### CODE REVIEW

File: data/wyntonfranklin.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    {
+      "name": "Check out my youtube channel",
+      "url": "https://www.youtube.com/@wftutorials1085",
+      "icon": "FaYoutube"
+    },
+     {
+      "name": "Learn how to code on my tutorials website",
+      "url": "https://app.wftutorials.com/",
+      "icon": "FaInternetExplorer"
+    },
```
Comment (Sindhuinti) [inline data/wyntonfranklin.json:32]: Thank u for ur contribution 😀 . Please remove the **comma in the line 32**  to pass all the checks

Comment (wyntonfranklin) [reply data/wyntonfranklin.json]: fixed

**Review by Sindhuinti — APPROVED** [review]
Looks Good 👍🏻

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**